### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-packer.git`](https://github.com/cisagov/skeleton-packer.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by mcdonnnj — APPROVED** [review]
🤖

File: src/requirements.yml [hunk 1]
```diff
@@ -3,8 +3,6 @@
   name: amazon_ssm_agent
 - src: https://github.com/cisagov/ansible-role-automated-security-updates
   name: automated_security_updates
-- src: https://github.com/cisagov/ansible-role-backports
```
Comment (mcdonnnj) [inline src/requirements.yml:6]: We should probably also remove the following that relied on this dependency:
https://github.com/cisagov/openvpn-packer/blob/68e28fc5edddc9f85343fa8cdebf989b9e39bc7e/src/aws.yml#L8-L9
https://github.com/cisagov/openvpn-packer/blob/68e28fc5edddc9f85343fa8cdebf989b9e39bc7e/src/aws.yml#L15-L28

**Review by mcdonnnj — CHANGES_REQUESTED** [review]

File: src/requirements.yml (hunk 1 above)
Comment (jsf9k) [reply src/requirements.yml:6]: All of that stuff is [already](https://github.com/cisagov/openvpn-packer/blob/lineage/skeleton/src/requirements.yml) [gone](https://github.com/cisagov/openvpn-packer/blob/lineage/skeleton/src/aws.yml), no?

**Review by dav3r — APPROVED** [review]
This all looks good.  My only question is whether there are any other feasible options to modify the ufw ssh rule so that it only allows in the Packer provisioner(s), now that we are removing the restriction to only allow ssh in from the `lo` interface?  I can't think of anything right now, but posing this question in case it spurs an idea in someone.

**Comment by jsf9k** [comment]
> This all looks good. My only question is whether there are any other feasible options to modify the ufw ssh rule so that it only allows in the Packer provisioner(s), now that we are removing the restriction to only allow ssh in from the `lo` interface? I can't think of anything right now, but posing this question in case it spurs an idea in someone.

I couldn't think of anything, other than creating an Ansible role that modifies the `sshd` config to only listen on `lo`.  I actually thought of doing that once before, and applying the role to all AMIs now that we only ever ssh in via SSM (and even that is very rare), but that would break some functionality that the RTA crew relies on.  I have not _removed_ `sshd` from our instances for similar reasons.

As I said in a comment, we should be OK since the security groups and network ACLs that apply to the OpenVPN instance already disallow external ssh access.

**Review by jmorrowomni — APPROVED** [review]
Based on previous conversations, LGTM

File: src/requirements.yml (hunk 1 above)
Comment (mcdonnnj) [reply src/requirements.yml:6]: Well that's peculiar I wasn't seeing it in my review page. Apologies and carry on!

**Review by mcdonnnj — APPROVED** [review]
:shipit:

**Pull request merged**